frontend/src/views/Login/components/PasswordStep/PasswordStep.tsx

```diff
@@ -127,6 +127,7 @@ export const PasswordStep = ({
             </div>
             <div className='lg:w-1/6 w-1/4 w-full mx-auto flex items-center justify-center min-w-[22rem] text-center rounded-md mt-4'>
                 <Button
+                    type="submit"
                     colorSchema="primary" 
                     variant="outline_bg"
                     onClick={async () => handleLogin()} 
```

Comment:
Can you please move this onClick to onSubmit in the form component. because this is not a good pattern to put both submit and onClick here